tests/ballerina-test/src/test/resources/test-src/taintchecking/connectors/sql-select-untainted-query.bal

```diff
@@ -0,0 +1,22 @@
+import ballerina.data.sql;
+
+public function main (string[] args) {
+    testSelectWithUntaintedQuery(args);
```

Comment:
What is the purpose of passing args into 'testSelectWithUntaintedQuery' function here.

Reply:
This is to make sure passing an untainted query will not yield any compile time errors. Basically, to make sure there is no negative impact of taint-analyzer for untainted operations.